### PR TITLE
Certain tasks excluded from stack updates vs creates

### PR DESCRIPTION
**Pull request by koneill94**
Tested through pipeline and it passed. Tested functionality manually by editing /etc/hosts file and git branch on the bastion then running a stack update. The update worked, /etc/hosts file was unchanged and the git code was not pulled again.

### CODE REVIEW

File: files/setup_bastion.yaml [hunk 1]
```diff
@@ -42,6 +42,17 @@
...
+
+    - name: Check if stack update or create and register variable
+      shell: if [[ $(openstack stack show "openshift-{{ osTenantName }}" -f value -c stack_status \
+                --os-auth-url "{{ osAuthUrl }}" \
+                --os-username "{{ openstackOpenshiftUsername }}" \
+                --os-project-name "{{ osTenantName }}" \
+                --os-password "{{ openstackOpenshiftPassword }}") = *CREATE* ]] ; then echo "True" ; else echo "False" ; fi
+      register: create_value
+
+    - set_fact: create="{{ create_value.stdout }}"
```
Comment (andywis) [inline files/setup_bastion.yaml]: Can we come up with a less ambiguous variable name than "create" ? e.g. is_stack_create

Comment (koneill94) [reply files/setup_bastion.yaml]: Changed create to stack_create

File: files/setup_bastion.yaml [hunk 2]
```diff
@@ -42,6 +42,17 @@
...
     extraGateway: __extra_gateway__
   tasks:
+
+    - name: Check if stack update or create and register variable
+      shell: if [[ $(openstack stack show "openshift-{{ osTenantName }}" -f value -c stack_status \
+                --os-auth-url "{{ osAuthUrl }}" \
+                --os-username "{{ openstackOpenshiftUsername }}" \
+                --os-project-name "{{ osTenantName }}" \
+                --os-password "{{ openstackOpenshiftPassword }}") = *CREATE* ]] ; then echo "True" ; else echo "False" ; fi
+      register: create_value
```
Comment (stevemul) [inline files/setup_bastion.yaml]: I don't like this variable name - its not clear enough what its for. Try something more descriptive.

Comment (stevemul) [reply files/setup_bastion.yaml]: Like stack status for example

Comment (benbacon) [reply files/setup_bastion.yaml]: Could this be changed so that it returns a Boolean value as opposed to echoing a string?

**Pull request merged**